Intelligent Defaults

davidpfahler opened the pull request:
We need some more intelligent defaults if we use wrapped directives. Here is what I mean:
- [x] `bp-navbar > bp-button[bp-tap]` should have `[bp-no-scroll]` enabled.
- [x] `bp-iscroll > bp-cell[bp-tap]` should `[bp-bound-margin=5]`.

Can we collect some more ideas here @davidpfahler?

boennemann commented:
- [x] `bp-button[bp-tap="to('foo',true)"]` should automatically have `.bp-button-back`.

davidnpma committed:
feat(tap): back button, bound margin and no scroll automagic

Intelligent defaults for tap directive.
1. back button
`bp-button[bp-tap="to('foo',true)"]` should automatically have
`.bp-button-back`.
2. no scroll
`bp-navbar > bp-button[bp-tap]` should have `[bp-no-scroll]` enabled.
3. bound margin
`bp-iscroll bp-cell[bp-tap]` should have `[bp-bound-margin=5]`.

davidnpma committed:
chore: merge

Merge branch 'master' into intelligentDefaults

* master:
  fix(a11y): don't read invisible search cancel button
  feat(a11y): add aria roles to directives

Conflicts:
	dist/bradypodion.js.map
